payone-adapter/src/main/java/com/commercetools/payment/payone/utils/impl/PayoneHandlePaymentHelperImpl.java

```diff
@@ -25,15 +26,16 @@ public PayoneHandlePaymentHelperImpl(SphereClient client) {
 
     @Override
     public CompletionStage<Payment> requestHandling(String paymentId) {
```

Comment:
I've not found any usage of this method as same as whole `PayoneHandlePaymentHelper` interface. It was inherited from old sources. Since it is an open library which is used in couple of projects, we _one does not simply removes an interface and the implementation_. We could mark it as `@Deprecated`